uktrade/data-hub-frontend
# Don't include all company contacts when saving the contact form

## Description of change

There is currently a live issue where new contacts cannot be saved for companies with many contacts or where a company has a contact with a large details field. This is due to the size of the payload. In order to resolve this issue, when creating/editing a contact the `company.contact` object is set to an empty array.

## Test instructions

It should be possible to add a contact with 50kb+ of text in the notes. It should also be possible to add or edit contacts associated with a company with many contacts (we have companies in prod with 100+ contacts).

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
